app/models/service_request.rb

```diff
@@ -426,10 +437,6 @@ def set_original_submitted_date
   end
 
   def set_ssr_protocol_id
-    if saved_change_to_protocol_id?
-      sub_service_requests.each do |ssr|
-        ssr.update_attributes(protocol_id: protocol_id)
-      end
-    end
+    self.sub_service_requests.each{ |ssr| ssr.update_attribute(:protocol, self.protocol) }
```

Comment:
In Rails 6 updating `protocol_id` will not increment the counter cache, so I went ahead and added this change which does.